lib/plugins/index.js

```diff
@@ -1,6 +1,9 @@
 'use strict';
 
-const _ = require('lodash');
+const h = require('highland'),
+  _map = require('lodash/map'),
+  _isFunction = require('lodash/isFunction'),
+  _cloneDeep = require('lodash/cloneDeep');
```

Comment:
Just a question, but any reason you're importing each separately rather than just `require('lodash')`? (since it's server-side)

Reply:
I'm starting to look the cost benefit of the how just importing `_` requires a running `Object.get` to run each time the function runs. I know V8 probably optimizes the calls so it doesn't really matter, I was just playing around.

ALSO it's on my list to remove lodash as much as possible (especially in the conversion of larger sections of the code to streams). It's nice to look at a file and get an initial glimmer at what it would take to refactor lodash out. I'll probably take this back to the original way, though.